src/openforms/js/components/form/map.js

```diff
@@ -62,6 +62,80 @@ const EDIT_FORM_TABS = {
         HIDDEN,
         CLEAR_ON_HIDE,
         {...IS_SENSITIVE_DATA, defaultValue: true},
+        {
+          type: 'number',
+          key: 'defaultZoom',
+          label: 'Default zoom level.',
+          description: 'Default zoom level for the leaflet map.',
+          validate: {
+            min: 1,
+            max: 18,
+          },
+        },
+        {
+          type: 'columns',
+          input: false,
+          label: 'starting location',
+          key: 'startPop',
+          tooltip: 'The Latitude and Longitude of the starting point of the map component.',
+          defaultValue: [{lat: '', lng: ''}],
+          columns: [
+            {
+              components: [
+                {
+                  label: 'Latitude',
+                  key: 'lat',
+                  input: true,
+                  type: 'number',
+                  decimalLimit: 7,
+                  delimiter: false,
+                  requireDecimal: true,
+                  tableView: false,
+                  inputFormat: 'plain',
+                  truncateMultipleSpaces: false,
+                  defaultValue: 52.1326332,
```

Comment:
aren't there a bunch of properties here that are the defaults and should be specified? Can we please limit the properties to only what's absolutely needed to improve readability?